source/_templates/installations/elastic/common/configure_kibana.rst

```diff
@@ -4,18 +4,18 @@
 
   # curl -so /etc/kibana/kibana.yml https://raw.githubusercontent.com/wazuh/wazuh/elastic-secured-3.10/extensions/kibana/7.x/kibana.yml
 
-The file ``/etc/kibana/kibana.yml`` has to be edited:
+Edit ``/etc/kibana/kibana.yml`` file:
 
 .. code-block:: yaml
 
     server.host: <kibana_ip>
-    elasticsearch.hosts: https://<elasticsearch_ip>:9200
+    elasticsearch.hosts: ["https://<elasticsearch_ip>:9200"]
```

Comment:
Are the characters `["` necessaries? If not, the less characters, the lesser user error possibilities.

Reply:
The characters are not necessary if there is only one Elasticsearch node listed.

In elasticsearch documentation it is presented like this:
"**elasticsearch.hosts:**
Default: [ "http://localhost:9200" ] The URLs of the Elasticsearch instances to use for all  your queries. All nodes listed here must be on the same cluster." 

https://www.elastic.co/guide/en/kibana/current/settings.html#settings

Should I change it back?